docs/pbc/all/product-information-management/202505.0/base-shop/install-and-upgrade/install-features/install-the-measurement-units-feature.md

```diff
@@ -0,0 +1,1057 @@
+---
+title: Install the Measurement Units feature
+description: The Measurement Units feature allows defining specific units of measure for products. The guide describes how to integrate the feature into your project.
+last_updated: Jun 16, 2021
+template: feature-integration-guide-template
+originalLink: https://documentation.spryker.com/2021080/docs/product-measurement-unit-feature-integration
+originalArticleId: a8fd28fc-83ac-4eda-ac81-81ec492970c1
+redirect_from:
+  - /2021080/docs/product-measurement-unit-feature-integration
+  - /2021080/docs/en/product-measurement-unit-feature-integration
+  - /docs/product-measurement-unit-feature-integration
+  - /docs/en/product-measurement-unit-feature-integration
+  - /docs/scos/dev/feature-integration-guides/201811.0/product-measurement-unit-feature-integration.html
+  - /docs/scos/dev/feature-integration-guides/202212.0/product-measurement-unit-feature-integration.html
+  - /docs/scos/dev/feature-integration-guides/202009.0/product-measurement-unit-feature-integration.html
+  - /docs/scos/dev/feature-integration-guides/202005.0/product-measurement-unit-feature-integration.html
+  - /docs/scos/dev/feature-integration-guides/202001.0/product-measurement-unit-feature-integration.html
+  - /docs/scos/dev/feature-integration-guides/201907.0/product-measurement-unit-feature-integration.html
+  - /docs/scos/dev/feature-integration-guides/201903.0/product-measurement-unit-feature-integration.html
+  - /docs/scos/dev/feature-integration-guides/201811.0/product-measurement-units-feature-integration.html
+  - /docs/pbc/all/product-information-management/202204.0/base-shop/install-and-upgrade/install-features/install-the-measurement-units-feature.html
+related:
+  - title: Install the Measurement Units Glue API
+    link: docs/pbc/all/product-information-management/page.version/base-shop/install-and-upgrade/install-glue-api/install-the-measurement-units-glue-api.html
+---
+
+## Install feature core
+
+### Prerequisites
+
+Install the required features:
+
+| NAME | VERSION |
+| --- | --- |
+| Cart | {{page.version}} |
+| Product | {{page.version}} |
+| Order Management | {{page.version}} |
+| Spryker Core | {{page.version}} |
+
+### 1) Install the required modules
```

Comment:
* We have to add a missing configuration for the Storage module under the `## Set up configuration`.
* Missing configuration: \Pyz\Zed\ProductMeasurementUnitStorage\ProductMeasurementUnitStorageConfig